Engine/TestStep.cs

```diff
@@ -844,6 +844,9 @@ internal static void UpgradeVerdict(this ITestStep step, Verdict newVerdict)
                 step.Verdict = newVerdict;
         }
 
+        [ThreadStatic]
+        internal static ITestStep currentlyExecutingTestStep = null;
```

Comment:
Let's add a comment stating what this is for.

Reply:
Added comments as requested